### DISCUSSION TITLE
Thanks for: Moving appointments between CalDAV calendars possible

### DISCUSSION THREAD

**Discussion by dktzde** [FossifyOrg/General-Discussion#467] [General]
FossifyCalendar is the first app where I can move existing entries between different calendars. This is very useful for me. Many thanks for that!

**Comment by pin88888** [comment]
I have just installed Fossify Calendar and I find it very very good!
Two question: 
1) How to synch events with KDE Calendar (very similar to Outlook but under Linux) ? 
It is only via CallDAV (or similar apps) or a sever possible? Actually my smartphone is connected to PC with KDE Connect; this should make possible synching data.
2) Does Fossify Calendar support  EXDATA (exceptions to a series recurring events)?

**Reply by Aga-C** [reply]
1. Generally, to synchronize CalDAV calendars, we rely on external apps like [Davx5](https://www.davx5.com/download). However, if you use KDE Connect, you may check, if you already have that calendar available. In Settings, turn on `CalDAV sync` and check if you can see your calendar. If not, the additional app is needed.
2. We support `EXDATE`, but I can't assure you that it's the full support according to iCalendar specification and how other apps set it. However, it should support most common use cases. If you see that some use case is not covered, please raise a [bug report](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml), attaching an example file.
